contracts/modules/delegates/IexecPoco1Delegate.sol

```diff
@@ -232,6 +232,7 @@ contract IexecPoco1Delegate is
         /**
          * Check orders authenticity
          */
+        //slither-disable-next-line uninitialized-local
```

Comment:
Not sure what could happen here

Reply:
Why do mean ? Slither reports the fact that `Matching memory ids;` is not initialized

Reply:
Yes, I don't see here if something bad could happen here

Reply:
Here nothing